amethyst_input/src/bindings.rs

```diff
@@ -15,7 +15,7 @@ where
     AC: Hash + Eq,
 {
     pub(super) axes: HashMap<AX, Axis>,
-    pub(super) actions: HashMap<AC, SmallVec<[Button; 4]>>,
+    pub(super) actions: HashMap<AC, SmallVec<[Vec<Button>; 4]>>,
```

Comment:
```suggestion
    pub(super) actions: HashMap<AC, SmallVec<[SmallVec<[Button; 2]>; 4]>>,
```
I believe that most actions will still only need a few buttons (mostly just 1 button)